### ISSUE TITLE
Few questions about the future plans

### ISSUE THREAD

**Issue by kasper3** [gentoo/dotnet#354]
* https://github.com/dotnet/source-build - would source-build simplify things for this repo and leave behind just the Portage packaging bits?

* Secondly, can we just use generic Linux binaries on Gentoo https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.0-download.md#net-core-200?

* Are there any efforts going on to get Gentoo OOTB official support and obsolete this separate effort?

**Comment by ArsenShnurkov** [comment]
1) thanks for cc, this is interesting and usefull documentation
2) somebody defenitely will create -bin packages, which use these binaries as is (similar to libreoffice-bin), but i personally have no interest in this
3) i didn't understood words about "separate effort". I don't know gentoo-based distribution which have all .net-tools preinstalled (I will prefer to create equo packages for Sabayon linux, because i use it)

**Comment by ArsenShnurkov** [comment]
who will pay for support?

**Comment by mpkh** [comment]
@ArsenShnurkov yes, there is the actual financial base but it's not used as salary for developers or something alike, it's for some hardware/network needs, etc.

I don't fully understand that issue, @kasper3 do you want to have all those binaries to be packaged? I absolutely see no problems with it except I'm not using dotnet at the moment and doubt if soon I will find time for that, contributions are very welcome.

**Comment by gsnerf** [comment]
It sounds like he wants to see something like gentoo ebuilds for the .net core ecosystem provided by the .NET Foundation

@kasper3 I'm not sure if they are willing to do something like that for a "niche distribution" like gentoo, there probably hasn't been any effort to ask them up until know though.

**Comment by ArsenShnurkov** [comment]
There a "non niche" distributions, like Sabayon and Calculate Linux. Or you can start some new distribution!

Your distribution will gather money for support, give them to gentoo foundation, and gentoo foundation will arrange support for .Net foundation.

**Comment by kasper3** [comment]
> gentoo ebuilds for the .net core ecosystem provided by the .NET Foundation

Yup, basically Microsoft is building packages for many distributions already (RedHat, Debian, openSUSE etc.). If Gentoo community shows interest, it might be added to the pool of official distribution. Microsoft will be responsible for releasing, keeping stuff up-to-date. Is it too unordinary in Gentoo ecosystem to have for example Oracle maintaining official OracleDB package, Google maintaining official Go package and so on?
